src/Swarm/App.hs

```diff
@@ -60,12 +67,34 @@ appMain seed scenario toRun cheat = do
           threadDelay 33_333 -- cap maximum framerate at 30 FPS
           writeBChan chan Frame
 
-      -- Run the app.
+      eventHandler <- case port of
```

Comment:
When the web interface is running, it is using a custom handleEvent handler to update a shared IORef. I made this optional to not interfere when the web interface is not activated, but perhaps the overhead is negligible, and in that case, I think we should always starts the web service. That way, there is no need to restart the game if the user forgot to use the `--web` argument, and the web view would be always available if needed.